---
published: true
category: spotlight
title: Human gut v2.0
emg:
  text: Browse Catalogue
  url: >-
    https://www.ebi.ac.uk/metagenomics/browse#genomes
layout: post
---
![Illustration of a catalogue with a taxonomic tree]({{site.baseurl}}/assets/media/images/posts/human-gut-v2-0/species_catalogue.png){:width="136px"} 
## Version 2 of the Unified Human Gastrointestinal Genome catalogue released
MGnify are excited to announce the release of [version 2 of the Unified Human Gastrointestinal Genome (UHGG) catalogue](https://www.ebi.ac.uk/metagenomics/genome-catalogues/human-gut-v2-0). This is an updated version of the catalogue published by [Almeida et al. _Nature Biotech_ (2021)](https://www.nature.com/articles/s41587-020-0603-3). We have added 5,878 new genomes from two studies ([PRJEB37358](https://www.ebi.ac.uk/ena/browser/view/PRJEB37358) and [PRJNA544527](https://www.ebi.ac.uk/ena/browser/view/PRJNA544527)), representing 129 new species. 

Notably, this includes the set of isolate genomes from the most recent human gut culture collection generated by [Poyet et al. _Nature Medicine_ (2019)](https://www.nature.com/articles/s41591-019-0559-3). This means that the catalogue now contains a total of 289,232 prokaryotic genomes from the human gut microbiome clustered into 4,744 species representatives. 

With the new genome additions, we have replaced 132 species representatives with better quality genomes (either a better quality MAG (metagenome assembled genome) or an isolate genome). To further increase the quality of the catalogue we have implemented the use of [GUNC](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02393-0) (a tool developed by the Bork group at EMBL) for genome quality filtering. Any singleton species (i.e., represented by only one strain) <90% complete flagged by GUNC as chimeric was excluded from the catalogue. Further details on the species replaced or excluded based on these criteria can be found in the associated [README](http://ftp.ebi.ac.uk/pub/databases/metagenomics/mgnify_genomes/human-gut/v2.0/README_v2.0.txt). We have also updated our genome accessions to use a standardised MGYG prefix, as well as a .version suffix to indicate an update to the genome sequence (such as for the removal of host contamination contigs).

![Species tree for the UHGG2.0 catalogue]({{site.baseurl}}/assets/media/images/posts/human-gut-v2-0/tree.png){:width="100%"} 

As for v1.0 we have also generated pan-genomes and an associated protein catalogue from all the genomes. However, we have replaced the use of [Roary](https://academic.oup.com/bioinformatics/article/31/22/3691/240757) with [Panaroo](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02090-4), due to its increased stringency for generating pan-genomes from MAGs. Protein-coding sequences were annotated with updated versions of eggNOG and InterPro.

For those who are relying on v1.0 for their ongoing projects do not worry: all the data from the former version is still available on [our FTP](http://ftp.ebi.ac.uk/pub/databases/metagenomics/mgnify_genomes/human-gut/v1.0/), and a mapping between the accession formats is provided in genomes-all_metadata.tsv.

Moving forward, we will continue to provide regular updates to our existing catalogues based on the availability and number of new genomes released, and technical advancements in the field. We hope you find these data useful and feel free to [provide us with any feedback](https://www.ebi.ac.uk/support/metagenomics) you have.

